app/migrations/Version20190205173350.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+final class Version20190205173350 extends AbstractMigration
+{
+    public function up(Schema $schema): void
+    {
+        $this->addSql('ALTER TABLE ideas_workshop_theme ADD position SMALLINT UNSIGNED NOT NULL');
+    }
+
+    public function postUp(Schema $schema)
```

Comment:
I had to manually set logical position instead of an update because otherwise position can't be managed through the admin edition